src/status_im/contexts/settings/wallet/saved_addresses/sheets/remove_address/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im.contexts.settings.wallet.saved-addresses.sheets.remove-address.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [name address customization-color]}]
+  (let [on-press-remove (rn/use-callback
+                         #(rf/dispatch [:wallet/delete-saved-address
+                                        {:address       address
+                                         :toast-message (i18n/label :t/saved-address-removed)}]))
+        on-press-cancel (rn/use-callback #(rf/dispatch [:hide-bottom-sheet]))]
```

Comment:
This could be extracted to a static `defn`, wdyt?

Reply:
Yep :)